spring-integration-core/src/main/java/org/springframework/integration/config/annotation/AbstractMethodAnnotationPostProcessor.java

```diff
@@ -72,6 +70,8 @@
 import org.springframework.util.ObjectUtils;
 import org.springframework.util.StringUtils;
 
+import org.aopalliance.aop.Advice;
+
 /**
```

Comment:
What do you mean here?
I have imports settings like `org.*` is coming after `org.springframework.*`
Or don't you like here that I have auto-optimization for imports?